x/auth/tests/abci.rs

```diff
@@ -0,0 +1,128 @@
+use auth::{AuthABCIHandler, GenesisState, Keeper};
+use gears::{
+    tendermint::types::time::timestamp::Timestamp,
+    utils::node::{init_node, GenesisSource, MockOptionsFormer},
+};
+use std::str::FromStr;
+
+use gears::{
+    params::{ParamsSubspaceKey, SubspaceParseError},
+    store::StoreKey,
+    types::address::AccAddress,
+    x::module::Module,
+};
+
+#[test]
+/// In this scenario, we test the initialization of the application and execute a few blocks
+fn test_init_and_few_blocks() {
+    let opt: MockOptionsFormer<
+        SubspaceKey,
+        AuthABCIHandler<SpaceKey, SubspaceKey, AuthModules>,
+        GenesisState,
+    > = MockOptionsFormer::new()
+        .abci_handler(AuthABCIHandler::new(Keeper::new(
+            SpaceKey::Auth,
+            SubspaceKey::Auth,
+            AuthModules::FeeCollector,
+        )))
+        .baseapp_sbs_key(SubspaceKey::BaseApp)
+        .genesis(GenesisSource::Genesis(GenesisState::default()));
+
+    let (mut node, _) = init_node(opt);
+
+    let app_hash = node.step(vec![], Timestamp::UNIX_EPOCH);
+    assert_eq!(
+        data_encoding::HEXLOWER.encode(app_hash),
+        "8d3663f81a98bec58a6d3a9f39c38469438bfecd8257dd335c8c047b933b08ad"
+    );
+
+    for _ in 0..100 {
```

Comment:
Please, add a method to node to skip steps. Something like: 
`node.skip_steps(100)`

Reply:
Ok

Reply:
Done

Reply:
The begin/end block will still need to run. So timestamps for every block will be needed